cms/static/js/spec/views/modals/move_xblock_modal_spec.js

```diff
@@ -53,11 +52,231 @@ define(['jquery', 'underscore', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpe
+                    };
+
+                    if (targetIndex !== undefined) {
+                        expectedData = _.extend(expectedData, {
+                            targetIndex: targetIndex
+                        });
+                    }
+
+                    // verify content of request
+                    AjaxHelpers.expectJsonRequest(requests, 'PATCH', '/xblock/', expectedData);
+
+                    // send the response
+                    AjaxHelpers.respondWithJson(requests, _.extend(responseData, {
+                        source_index: sourceIndex
+                    }));
+                }
+            };
+
+            moveXBlockWithSuccess = function(requests) {
+                var sourceIndex = 0;
+                enableMove(sourceIndex);
+                sendMoveXBlockRequest(requests, sourceLocator, targetParentLocator);
+                expect(modal.movedAlertView).toBeDefined();
+                expect(modal.movedAlertView.options.title).toEqual(getConfirmationFeedbackTitle(sourceDisplayName));
+                expect(modal.movedAlertView.options.titleHtml).toEqual(
+                    getConfirmationFeedbackTitleHtml(targetParentLocator)
+                );
+                expect(modal.movedAlertView.options.messageHtml).toEqual(
+                    getConfirmationFeedbackMessageHtml(
+                        sourceDisplayName,
+                        sourceLocator,
+                        sourceParentLocator,
+                        sourceIndex
+                    )
+                );
+            };
+
+            it('disabled by default', function() {
+                expect(isMoveEnabled(0)).toBeFalsy();
+            });
+
+            it('can not move is disabled state', function() {
+                var requests = AjaxHelpers.requests(this);
+                expect(isMoveEnabled(0)).toBeFalsy();
+                sendMoveXBlockRequest(requests, sourceLocator, targetParentLocator);
+                expect(modal.movedAlertView).toBeNull();
+            });
+
+            it('moves an xblock when move button is clicked', function() {
+                var requests = AjaxHelpers.requests(this);
+                moveXBlockWithSuccess(requests);
+            });
+
+            it('undo move an xblock when undo move button is clicked', function() {
+                var sourceIndex = 0,
+                    requests = AjaxHelpers.requests(this);
+                moveXBlockWithSuccess(requests);
+                modal.movedAlertView.undoMoveXBlock({
+                    target: $(modal.movedAlertView.options.messageHtml.text)
+                });
+                AjaxHelpers.respondWithJson(requests, {
+                    move_source_locator: sourceLocator,
+                    parent_locator: sourceParentLocator,
+                    target_index: sourceIndex
+                });
+                expect(modal.movedAlertView.movedAlertView.options.title).toEqual(
```

Comment:
We should verify the rendered message instead of using `modal.movedAlertView.movedAlertView.options.title`